### PR TITLE
Add ajv 8 validator to core tests and to playground w/ l12n

### PR DESCRIPTION
**Pull request by heath-freenome**
### Reasons for making this change

Reimplement https://github.com/rjsf-team/react-jsonschema-form/pull/2189 in a much simpler manner in v5 beta

- Updated `core` to add `@rjsf/validator-ajv8` as a devDependency and replicated the ajv6 tests for ajv8 with message text changes
- Updated `playground` to add `@rjsf/validator-ajv8` and `ajv-i18n` to support changing to validators
  - Added the default `AJV8` validator and a `AJV8_es` (using `ajv-i18n`) validator to the validators list
- Updated the `README.md` for `utils` to make it describe utils rather than the ajv6 validator
- Updated descriptions of tests in `validator-ajv6` to properly describe some test cases
- Updated `validator-ajv8` to add support for a localizer as follows:
  - Added a new `Localizer` type to for the localizer function
  - Updated the `customizeValidator()` function to take an optional second `localizer` parameter which it passes into the `AJV8Validator` constructor
  - Updated the `AJV8Validator` class to add an internal `localizer: Localizer` variable that is initialized from the new, optional `localizer` parameter on the constructor
  - Updated the `validateFormData()` function to call the `this.localizer()` with the `this.ajv.errors` if a function is provided
  - Updated the tests for the `customizeValidator()` and `AJV8Validator` to test the new behavior
- Updated the `validation.md` documentation for this new feature
- Updated the `CHANGELOG.md` for these changes

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [x] **I'm adding or updating code**
  - [x] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [x] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [x] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/master/CHANGELOG.md) with a description of the PR
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

Screenshot 2022-09-06 at 4 00 29 PM

### CODE REVIEW

**Comment by heath-freenome** [comment]
@IgorLesnevskiy

**Comment by IgorLesnevskiy** [comment]
looks good, thank you!

**Review by nickgros — APPROVED** [review]
Awesome! Just a few docs suggestions

File: docs/usage/validation.md [hunk 1]
```diff
@@ -334,9 +334,10 @@ First, there are many things to be aware of related to internal migration from A
 One big difference is that Ajv 8 dropped support for any JSON Schema version before draft-06.
 So if your schema is using an older format, you have to either upgrade it or stick with the `@rjsf/validator-ajv6`.
 
-The `ajvOptionsOverrides` for the Ajv 8 validator are the ones supported by that version and not previous versions.
-Secondly, the data formats previously provided in Ajv 6 now need to be added explicitly using the `ajv-formats` package.
+The `ajvOptionsOverrides` for the Ajv 8 validator are the ones supported by that version and not the Ajv 6 validator.
+Second, the data formats previously provided in Ajv 6 now need to be added explicitly using the `ajv-formats` package.
 A new `ajvFormatOptions` option is available on the `customizeValidator()` API to be able to configure this.
+Finally, the Ajv 8 validator support the localization of error messages.
```
Comment (heath-freenome) [inline docs/usage/validation.md:340]:
```suggestion
Finally, the Ajv 8 validator supports the localization of error messages.
```

**Pull request merged**